parser/src/namespace.rs

```diff
@@ -35,7 +35,7 @@ impl<'input> Namespace<'input> {
 
     /// The parent namespace.
     pub fn parent(&self) -> Option<&Namespace<'input>> {
-        self.parent.as_ref().map(|x| &**x)
```

Comment:
I think it's because of this change.

Which actually implies that it is a false-positive in clippy?

Reply:
Not true still

Segmentation fault